com.github.Return-to-the-Roots.yaml

```diff
@@ -0,0 +1,91 @@
+app-id: com.github.Return-to-the-Roots
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+command: s25client
+rename-desktop-file: s25rttr.desktop
+rename-icon: s25rttr
+rename-appdata: s25rttr.metainfo.xml
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=pulseaudio
+  - --share=network
+  - --device=dri
+
+modules:
+  - name: boost
+    buildsystem: simple
+    build-commands:
+      - ./bootstrap.sh --prefix=${FLATPAK_DEST} --with-libraries=test,locale,iostreams,filesystem,program_options,regex,system,chrono
+      - ./b2 -j${FLATPAK_BUILDER_N_JOBS} install variant=release cxxstd=17 --layout=system
+    sources:
+      - type: archive
+        url: https://boostorg.jfrog.io/artifactory/main/release/1.69.0/source/boost_1_69_0.tar.bz2
+        sha256: 8f32d4617390d1c2d16f26a27ab60d97807b35440d45891fa340fc2648b04406
+
+  - name: boost-nowide
+    buildsystem: cmake-ninja
+    config-opts:
+      - -DNOWIDE_SYSTEM_INCLUDE=ON
+    sources:
+      - type: git
+        url: https://github.com/boostorg/nowide.git
+        tag: v11.1.4
+        commit: b4c10155171b782fc858ae1aadcfdbe34e4940d4
+
+  - name: SDL2
```

Comment:
sdl 2.0.16 is in the runtime. isn't it enough?

Reply:
I got CMake errors about it not being found.

Reply:
> CMake Error at /usr/share/cmake-3.22/Modules/FindPackageHandleStandardArgs.cmake:230 (message):
  Could NOT find SDL2 (missing: SDL2_SDL2MAIN_LIBRARY) (found version
  "2.0.16")
Call Stack (most recent call first):
  /usr/share/cmake-3.22/Modules/FindPackageHandleStandardArgs.cmake:594 (_FPHSA_FAILURE_MESSAGE)
  cmake/Modules/FindSDL2.cmake:87 (find_package_handle_standard_args)
  external/s25edit/SGE/CMakeLists.txt:6 (FIND_PACKAGE)

Reply:
I only have a guess, but could it be that its the same issue like in nixpkgs, that SDLmain.a was removed accidently? see https://github.com/Return-To-The-Roots/s25client/pull/1511

Reply:
We generally want dynamic linking for Flatpak packages too, as it improves deduplication for updates. From downstream perspective, CMake shouldn't bail out completely only because static library is missing.

Reply:
I experimented with it. It does not only check for the static version, it also requires it to build.